druid/src/window.rs

```diff
@@ -139,6 +141,25 @@ impl<T: Data> Window<T> {
     pub fn paint(&mut self, paint_ctx: &mut PaintCtx, data: &T, env: &Env) {
         let visible = Rect::from_origin_size(Point::ZERO, self.size);
         paint_ctx.with_child_ctx(visible, |ctx| self.root.paint(ctx, data, env));
+
+        paint_ctx.z_ops.sort_by_key(|k| k.z_index);
```

Comment:
sort of silly but I might do the sorting after we take the items out of the context? It just feels very slightly easier to read.